net/tcp/tcp_netpoll.c

```diff
@@ -230,7 +230,12 @@ int tcp_pollsetup(FAR struct socket *psock, FAR struct pollfd *fds)
   info = conn->pollinfo;
   while (info->conn != NULL)
     {
-      DEBUGASSERT((fds->events & info->fds->events) != 0);
+      if ((fds->events & info->fds->events) != 0)
```

Comment:
if it's valid, why not remove the warning too

Reply:
The warning is used to catch this situation.

Reply:
but if this is a valid operation, why need output the warning message to confuse user.

Reply:
let us remove this warning treat as normal case

Reply:
@anchao OK